paqr/publish.py

```diff
@@ -0,0 +1,22 @@
+from paqr.paq import validate_paq
+from paqr.docker_utils import *
+from paqr.gcloud_utils import *
+import argparse
+from paqr import qpr
+
+
+def main(args):
+    config = validate_paq(args.paq_dir)
+    docker_build(config['name'], args.paq_dir)
+    gcr_id = docker_tag(config['name'])
+    docker_push(gcr_id)
+    gcloud_deploy(gcr_id, config)
+    qpr.publish(config)
```

Comment:
Insofar as qpr.publish will behave the way the comment suggests, it seems this flow makes no provision for pushing an update

Reply:
Good call, we'll need to handle this case. Where we will need to go with this, if we make this sort of thing public, is to require users to supply their API Key when they are publishing models. This way we can check if someone is the owner of a pre-existing model and confirm with them that they want to overwrite it